### ISSUE TITLE
XML File Size Supported?

### ISSUE THREAD

**Issue by ecjacobson** [reportunit/reportunit#17]
My TestResult.xml is 25MB.  Is that supported?  reportunit appears to hang.  I see "78823 tests processed..." as the last thing listed in the console.  The HTML files never get created.

**Comment by anshooarora** [comment]
After it processes tests, it will take a few more minutes to actually write everything to the file and save. But it will work.  There are some fixes planned for performance issues, shouldn't take so long in the upcoming versions.

**Comment by ecjacobson** [comment]
I've waited > 3 hours.

**Comment by ecjacobson** [comment]
I can probably send you my testresults.xml if you want to give it a go.

**Comment by anshooarora** [comment]
Yes that would help.  Its a big file not sure you will be able to email, can you upload it and send me the link.  You can email me at anshooarora@gmail.com.

**Comment by ecjacobson** [comment]
I sent you an email with a link to the testResult file.  The HTML pages did eventually work (kind of).  It took maybe 10 hours.  Also chrome crashed on attempt to browser the tests from  TestResult_BankClient.html.

**Comment by ecjacobson** [comment]
Did you get the 25MB test file (testresults.xml) okay?

**Comment by anshooarora** [comment]
Yes, thank you for that.  I haven't had a chance to test yet though.  I will do it tonight or tomorrow morning.  Will update you soon.

**Comment by ecjacobson** [comment]
Thanks, man

**Comment by anshooarora** [comment]
I tested the file last night and have made changes to the parser.  This process now takes a second as compared to several minutes.  Still working on the markup generation, its still very slow.

Also, you must note that the result HTML file will be approx 25-30MB in size and will be quite slow and laggy to view because the js will have to run for thousands of elements.

**Comment by ecjacobson** [comment]
got it.  I appreciate your attention.

**Comment by anshooarora** [comment]
The parser will be completely revamped.  I will launch the new version (in beta) in a few weeks.  There will be a lot of changes so would take some time..

**Comment by anshooarora** [comment]
I was able to create most of the NUnit parser which is completely Linq and it took 18 mins to process. The resulting file was 50MB - I can share it with you.

**Comment by BADF00D** [comment]
Hi,
using ReportUnit to generate a report of approx. 5500 tests took about 4-5min with version 1.2. With the master from September 2017 (version v1.50.0 in GUI) it took about 2 sec. So there is a huge performance boost in the newer version. 
The resulting html from version 1.2 was pretty slow (it took about 10 sec until a filter was applied and the GUI was rendered). With master from September 2017, the filters seem not wo work at all.
